src/components/Modals/Send/views/Address.tsx

```diff
@@ -49,16 +49,15 @@ export const Address = () => {
 
   const handleNext = useCallback(() => navigate(SendRoutes.Details), [navigate])
 
-  const handleClick = useCallback(
-    () =>
-      navigate(SendRoutes.Select, {
-        state: {
-          toRoute: SelectAssetRoutes.Search,
-          assetId: asset?.assetId ?? '',
-        },
-      }),
-    [navigate, asset],
-  )
+  const handleBackClick = useCallback(() => {
+    setValue(SendFormFields.AssetId, '')
```

Comment:
That and the below was wrong - if we're going back to `select`, we obviously don't have an `AssetId` selected anymore.